### ISSUE TITLE
can not override accept-encoding header in lib/web.php

### ISSUE THREAD

**Issue by m0x3** [bcosca/fatfree#1260]
I need NOT send gzip,deflate, cause it makes troubles on other side, but f3 always sends gzip,deflate.

**Comment by ikkez** [comment]
Yes I see it's not editable. We'll have a look to fix that. In the meantime you try to switch the request engine and try it with socket or stream engine instead of curl:

```php
$web = \Web::instance();
$web->engine('stream');
$web->request('https://domain.com');
```

These do accept the same $options array as curl.

ref.:
https://github.com/bcosca/fatfree-core/blob/cdd878db018280369b643fce62f70e55044f9ac4/web.php#L362
